.github/workflows/ci.yml

```diff
@@ -165,7 +165,9 @@ jobs:
 
       - name: Run API codegen
         if: steps.cache-api-code.outputs.cache-hit != 'true'
-        run: make api-codegen
+        run: |
+          poetry install --all-extras
+          make api-codegen
```

Comment:
Need to install reader package.

![image](https://github.com/ansys/pyfluent/assets/106588300/f37aa142-a377-41ef-996d-e155a738b043)

https://github.com/ansys/pyfluent/actions/runs/9889959134/job/27317176855?pr=3058